app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for human in people:
+        Person.people[human["name"]] = Person(human["name"], human["age"])
+        if human.get("husband"):
+            wife = Person.people[human["name"]]
+            husband = Person.people[human["husband"]]
+            wife.husband = husband
+            husband.wife = wife
+    return [person for name, person in Person.people.items()]
```

Comment:
This works wrong
Because:
1. You must return only created list, not all of items
2. You must set wife also(for this you must check if human.get("wife")
3. Person.people[human["name"]] = Person(human["name"], human["age"])
Is so strange because it do the same inside init method
4. Use 2 loops: 1 - create all person, 2 - set wife/husband

Reply:
Thank you.

For the p.2 I assumed that if someone in the list has husband, I cross-link wife-husband and therefore I don't need to check "wife" key